site/src/util/workspace.test.ts

```diff
@@ -47,7 +47,7 @@ describe("util > workspace", () => {
       [
         "2022-06-02T14:56:34Z",
         {
-          deadline: "2022-06-02T16:26:34Z",
+          deadline: "2022-06-02T18:56:34Z",
```

Comment:
**Review**:

It was previously 90 minutes, now it's 4 hours. So this adds 2.5 hours.